index.html

```diff
@@ -27,8 +27,8 @@
 					<div class="nav__item">About</div>
 					<div class="nav__item">Partner</div>
 					<div class="nav__button">
-						<button class="login">Login</button>
-						<button class="register">Register</button>
+						<button class="white">Login</button>
+						<button class="orange">Register</button>
 					</div>
```

Comment:
orange and white are colors
it is a bad approach to name class like colors and add some style to them. 

rename back

Reply:
you can add more then 1 class to element, like this:
`<button class="login white">Login</button>`